### PR TITLE
Faq: system common operations

### PR DESCRIPTION
**Pull request by tinmarino**
Add common operations as specified here #3109 (3/5).
Waiting for these 3 before pushing the last 2 (Hash, Object and Syntax) and searching for some more Perlish common questions on SO.

### CODE REVIEW

File: doc/Language/faq.pod6 [hunk 1]
```diff
@@ -299,6 +299,69 @@ dependencies within a single compilation unit (e.g., file) are possible
 through stubbing. Therefore another possible solution is to move
 classes into the same compilation unit.
 
+=head2 System: How can I work with file and folder
+
```
Comment (JJ) [inline doc/Language/faq.pod6]: Not a well formulated question. Also, there's no issue requesting it, there's a [whole tutorial page for input/output](https://docs.raku.org/language/io), as well as the pages of the respective utilities. Let's please keep FAQs for FAQs, and [in this issue](https://github.com/Raku/doc/issues/3109) most of them were already covered.

File: doc/Language/faq.pod6 [hunk 2]
```diff
@@ -299,6 +299,69 @@ dependencies within a single compilation unit (e.g., file) are possible
...
+say 'is a file' if 'file.txt'.IO.L<f|/type/IO::Path#method_f>;
+say 'is a directory'  if 'dir'.IO.L<d|/type/IO::Path#method_d>;
+
+=item Write: 'file.txt'.IO.L<spurt|/type/IO::Path#method_spurt>: 'I ❤ 🦋'; Use the C<:append> adverb to append.
+
+=item Open: my $fh = 'file.txt'.IO.L<open|/type/IO::Path#method_open>;
+
+File handles are powerfull, read the L<overview|/language/io>, L<guide|/language/io-guide> or L<IO::Path|/type/IO::Path>. As usual, there are hundreds of ways to code with them, find yours!
+
+=head2 System: How can I recursively list file (find command)
```
Comment (JJ) [inline doc/Language/faq.pod6]: Not clear what you want to say, but grammar is incorrect. Also, not a FAQ if there's a simple module that can fix that.

File: doc/Language/faq.pod6 [hunk 3]
```diff
@@ -299,6 +299,69 @@ dependencies within a single compilation unit (e.g., file) are possible
...
+
+Or use File::Find module:
+
+=begin code
+use File::Find;
+.say for find dir => '.', name => /'.txt' $/;
+=end code
+
+=head2 System: How can I read from STDIN
+
```
Comment (JJ) [inline doc/Language/faq.pod6]: [This is literally the first result in Google](https://stackoverflow.com/questions/13893392/how-do-i-read-from-stdin-in-rakudo-perl6). Don't think it's a FAQ either.

**Comment by tinmarino** [comment]
It was mostly IO related stuff, so be better in IO. But I liked the

```raku
=head2 System: How can I check the execution operating system

L<$*KERNEL|/language/variables#$*KERNEL> gives information on the underlying kernel.
To check is running on windows use L<$*DISTRO|/language/variables#$*DISTRO>.is-win.
```

That appeared in variables.

**Comment by JJ** [comment]
> It was mostly IO related stuff, so be better in IO. But I liked the
> 
> ```perl6fe
> =head2 System: How can I check the execution operating system
> 
> L<$*KERNEL|/language/variables#$*KERNEL> gives information on the underlying kernel.
> To check is running on windows use L<$*DISTRO|/language/variables#$*DISTRO>.is-win.
> ```
> 
> That appeared in variables.

As said in other places, if you think that's not well explained in the place $*KERNEL is referenced, please add some words to that effect. Reference should be useful and also answer all kind of questions.

**Pull request closed without merging**